extension-manifest-v2/manifest.json

```diff
@@ -3,8 +3,8 @@
 	"author": "Ghostery",
 	"name": "__MSG_name__",
 	"short_name": "Ghostery",
-	"version": "8.9.14",
-	"version_name": "8.9.14",
+	"version": "8.10.0",
```

Comment:
I would not bump the version in feature PR, it should be done in a separate PR with its own commit message (and then you have a clean commit, which you can tag).